package.json

```diff
@@ -119,19 +122,21 @@
     "world-calendars": "^1.0.3"
   },
   "devDependencies": {
-    "@babel/core": "^7.25.2",
-    "@babel/plugin-transform-modules-commonjs": "^7.23.3",
-    "@babel/preset-env": "^7.23.9",
     "@biomejs/biome": "1.8.3",
     "amdefine": "^1.0.1",
-    "babel-loader": "^9.1.3",
+    "assert": "^2.1.0",
```

Comment:
where is `assert` used?

Reply:
Good eye. It might be the result of a mistake when resolving the merge conflicts.
I uninstalled it in https://github.com/plotly/plotly.js/pull/6909/commits/9f0267b815ac7f1a09fb4da331e752822dd71254.